core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -486,7 +488,9 @@ private[kafka] abstract class Acceptor(val socketServer: SocketServer,
                                        apiVersionManager: ApiVersionManager)
   extends Runnable with Logging {
 
-  private val metricsGroup = new KafkaMetricsGroup(this.getClass)
+  private val metricsPackage = "kafka.network"
+  private val metricsClassName = "Acceptor"
+  private val metricsGroup = new KafkaMetricsGroup(metricsPackage, metricsClassName)
```

Comment:
It seems this `metricsGroup` is useless since there is already a `backwardCompatibilityMetricGroup`